crowd-rt-forecast/redeploy.R

```diff
@@ -0,0 +1,78 @@
+library(covid.ecdc.forecasts)
+library(data.table)
+library(dplyr)
+library(rsconnect)
+library(here)
+
+# if today is not Monday, set submission date to last monday
+submission_date <- latest_weekday()
+
+check_dir(here("crowd-rt-forecast", "data-raw"))
+saveRDS(submission_date,
+        here("crowd-rt-forecast", "data-raw", "submission_date.rds"))
+first_forecast_date <- as.character(as.Date(submission_date) - 16)
+
+# copy Rt data into app
+obs <-
+  fread(
+    here("rt-forecast", "data", "summary", "cases", submission_date, "rt.csv")
+    ) %>%
+  rename(value = median, target_end_date = date) %>%
+  mutate(target_type = "case", target_end_date = as.Date(target_end_date)) %>%
+  filter(target_end_date <= (as.Date(first_forecast_date) + 7 * 6)) %>%
+  arrange(region, target_type, target_end_date) %>%
+  rename(location_name = region) %>%
+  select(-strat, -type, -mean, -sd)
+
+fwrite(obs, here("crowd-rt-forecast", "data-raw", "observations.csv"))
+
+# copy Rt samples and fit data as well -----------------------------------------
+copyrt <- function(origin_dir, target_dir, locations, date) {
+  for (location in locations) {
+    origin_folder <- here(origin_dir, location, date)
+    target_folder <- here(target_dir, location)
+    
+    if (dir.exists(origin_folder)) {
+      check_dir(target_folder)
+      file.copy(from = origin_folder, to = target_folder, recursive = TRUE)
+      
+      # read in epinow2 fit and thin
+      fit <- readRDS(here(target_folder, date, "model_fit.rds")) %>%
+        shredder::stan_slice(seq(1, 450, by = 450/25), inc_warmup = FALSE)
```

Comment:
I thought we diced to use the hacked version of the code as this package was a mess. Don't mind either way.

Reply:
I don't totally understand what the Rt model bits are only needed on redeployment but I assume it makes sense.

Reply:
I wasn't able to get it done without shredder and that works. 
oh but need to add shredder to the dependencies...